docs/reference/cli.md

```diff
@@ -236,19 +320,19 @@ Quantiles [1%, 25%, 50%, 75%, 99%]: [448, 448, 448, 448, 448]
 Searches an index with ID `--index` and returns the documents matching the query specified with `--query`.
 More details on the [query language page](query-language.md).
 The offset of the first hit returned and the number of hits returned can be set with the `start-offset` and `max-hits` options.
-It's possible to override the default search fields `search-fields` option to define the list of fields that Quickwit will search into if the user query does not explicitly target a field in the query. Quickwit will return snippets of the matching content when requested via the `snippet-fields` options.
```

Comment:
you lost the `snippets` I think because we did not put that text in the cli_doc_ext.toml.